### ISSUE TITLE
femto upgrade

### ISSUE THREAD

**Issue by brettrowberry** [Zaid-Ajaj/Femto#69]
Would it make sense for there to be a `femto upgrade` that basically calls `dotnet paket upgrade && yarn upgrade`?

**Comment by MangelMaxime** [comment]
For me, Femto is for working with the `npm` dependencies I don't think I want it to touch my .NET dependencies.

Currently, you can update your .NET dependencies using paket or dotnet. Then you can run `femto resolve` which will resolve and install the correct version of  `npm` dependencies based on the installed .NET dependencies.

Also femto would need to detect if you are using paket or dotnet to manage your .NET dependencies and call the corresponding one like it does for npm/yarn.

**Comment by Zaid-Ajaj** [comment]
@MangelMaxime Actually Femto does interact with your dotnet dependencies when you run `femto install` or `femto uninstall`. 

@brettrowberry An `upgrade` command would be a _nice to have_ though not really needed since you can simply run paket upgrade yourself and then run `femto --resolve` afterwards. NOT `yarn upgrade` because we don't know which packages will be required by the new versions of the dotnet dependencies, so just `femto --resolve` and let it do the magic.

A PR would be really appreciated :pray: a lot of the work is already done here (finding paket and invoking it) so only adding the command line arg, executing paket upgrade and then `femto --resolve`

**Comment by MangelMaxime** [comment]
@MangelMaxime Yes, it reads them from what I remember. My wording was a bit wrong I agree :)

**Comment by brettrowberry** [comment]
That makes sense. So a `femto upgrade` would really be a `paket upgrade` and `femto --resolve`.

**Comment by brettrowberry** [comment]
And it would only be possible if paket is used, not nuget.

**Comment by Zaid-Ajaj** [comment]
@brettrowberry Yes only for paket. Unless of course you are feeling extremely adventurous and want to implement the functionality for Femto :smile: No just kidding, this would be like opening a pandora's box

**Comment by MangelMaxime** [comment]
It would be strange if `femto install` and `femto uninstall` works with paket and dotnet but not `femto update` no?

In theory, we just need to execute `dotnet update` against the specified fsproj or is there something else I don't see?

**Comment by Zaid-Ajaj** [comment]
@MangelMaxime I don't think dotnet has an `update` command like paket does and it is only per-project.

**Comment by MangelMaxime** [comment]
Oops didn't know that. 😅

Then we should just warn that it is not supported in the console because the user doesn't use paket. Just so he knows why :) <- Thinking about future me 😂

**Comment by Zaid-Ajaj** [comment]
lol :joy: yeah absolutely: if we can't find paket then we simply say this is a paket-only feature

**Comment by brettrowberry** [comment]
Anybody want to pair on this?
